docs/install/helm.md

```diff
@@ -114,23 +117,28 @@ ako-1593523840	avi-system	1       	2024-08-04 13:44:31.609195757 +0000 UTC	    d
 
 *Step3*
 
-Get the values.yaml for the AKO version 1.14.1 and edit the values as per the requirement.
+Get the values.yaml for AKO version 1.14.1 and edit the values as per the requirement.
 
 ```
 helm show values oci://projects.packages.broadcom.com/ako/helm-charts/ako --version 1.14.1 > values.yaml
 
 ```
 *Step4*
 
-Upgrade the helm chart
+Upgrade the Helm chart:
+
+```
+helm upgrade ako-1593523840  oci://projects.packages.broadcom.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.14.1 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.username=<avi-ctrl-username> --set avicredentials.password=<avi-ctrl-password> --namespace=avi-system --dependency-update
 
 ```
-helm upgrade ako-1593523840  oci://projects.packages.broadcom.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.14.1 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.password=<username> --set avicredentials.username=<username> --namespace=avi-system
 
+Upgrade the secondary AKO:
+```
+helm upgrade  <secondary-ako-chart-name>  oci://projects.packages.broadcom.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.14.1 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.username=<avi-ctrl-username> --set avicredentials.password=<avi-ctrl-password> --namespace=avi-secondary-system --set ako-crd-operator.enabled=false --dependency-update
 ```
 
 ***Note***
-1. In multiple AKO deployment scenario, all AKO instances should be on the same version.
+1. In a multiple AKO deployment scenario, all AKO instances should be on the same version.
```

Comment:
Should we mention the same ako-crd-operator note here. "In a given cluster, there will only one ako-crd-operator instance running"?